rectools/metrics/auc.py

```diff
@@ -307,6 +320,8 @@ class PartialAUC(_AUCMetric):
         until the model has non-zero scores for the item in item-item similarity matrix. So with
         small `K` for neighbours in ItemKNN and big `K` for `recommend` and AUC based metric you
         will still get an error when `insufficient_handling` is set to `raise`.
+    debias_config : DebiasConfig, default None
```

Comment:
```suggestion
    debias_config : DebiasConfig, optional, default None
```

Reply:
I won't repeat it anymore, please fix everywhere

Reply:
Fixed